garrysmod/gamemodes/base/entities/entities/lua_run.lua

```diff
@@ -1,10 +1,14 @@
 
 ENT.Type = "point"
 ENT.DisableDuplicator = true
+ENT.MapCreationID = -1
 
 AccessorFunc( ENT, "m_bDefaultCode", "DefaultCode" )
 
 function ENT:Initialize()
+
+	self.MapCreationID = self:MapCreationID()

```

Comment:
Good shout, but it wouldn't actually override the method (because the entity functions have priority over any values set on the entity.)

So the commit actually errors as it is when trying to concat the values on line 45.